packages/editor/src/components/post-taxonomies/style.scss

```diff
@@ -1,6 +1,5 @@
 .editor-post-taxonomies__hierarchical-terms-list {
 	max-height: 14em;
-	overflow: auto;
```

Comment:
We can't remove that because the list has a limited height, it needs to be scrollable.

I was able to fix it by using negative margin with the width of the focus outline.

```css
margin-left: -5.5px;
padding-left: 5.5px;
margin-top: -5.5px;
padding-top: 5.5px;
```

This would replace the `padding-left: 2px;` workaround in line 5/6. 5.5 Pixel is the result of `$border-width * 4 + $border-width-focus` as used [here](https://github.com/WordPress/gutenberg/blob/b9de73ddb58b6b88faa4846d3f8294c815c9d4df/packages/components/src/checkbox-control/style.scss#L30).

Though, I noticed in Chrome that it has some rendering issues with the half pixel value:

![image](https://user-images.githubusercontent.com/617637/88478961-678d5080-cf4c-11ea-983f-4c417ac1c620.png)

The top/left focus outline of the first input field is smaller than the other fields.

Reply:
Good catch, how did I miss that :)

I applied the suggestion and I think the half-pixel issue is not unique to this particular case, it's something @MichaelArestad is trying to fix separately across the UI.

Reply:
@youknowriad I have tried two methods. Neither works right now. I ended up having to shelve it for now until either PostCSS updates to work properly (was failing when encountering `calc()` within a box shadow or someone comes up with another idea. :/ It was just taking too much time. I will definitely revisit as the random border widths on 1x devices make the whole editor look bad.

Reply:
Maybe we should just abandon the 1.5 borders.

Reply:
@youknowriad That's certainly the simplest solution.